### DISCUSSION TITLE
Add support for OnDevice Conversion measurement for Flutter platform

### DISCUSSION THREAD

**Discussion by puf** [firebase/flutterfire#10895] [Feature request]
Google Analytics has the ability to measure iOS Ads conversions. Per documentation[1], it looks like supported platform is iOS (Swift/Obj-C) and Unity. For consistency it'd be good to have this available in the Flutter SDK too.

[1] https://firebase.google.com/docs/tutorials/ads-ios-on-device-measurement/step-3#call_the_api

**Comment by skol101** [comment]
Same as https://github.com/firebase/flutterfire/discussions/10990
